### ISSUE TITLE
Postgres ArrayField in ModelSchema throws KeyError

### ISSUE THREAD

**Issue by abdelrahmenAyman** [vitalik/django-ninja#353]
I have a model `Project` containing an ArrayField zip_file_urls, this field is imported from `django.contrib.postgres.fields`.
Then I defined my `ProjectSchema` as the following:
```py
class ProjectSchema(ModelSchema):
    class Config:
        model = Project
        model_fields = ["project_id", "zip_file_urls"]
```
Then I use this schema to validate the following endpoint:
```py
@api.post("/projects")
async def create_project(request, project: ProjectSchema):
    return project
```
When I try to access the docs to try the endpoint it throws a `KeyError: ArrayField`. My workaround was to remove the `zip_file_urls` from the model fields, and define a field on the schema `zip_file_urls: list[HttpUrl]`.
Is there something that I am missing that can help me add the field to model_fields normally?

**Comment by dhendry** [comment]
+1 on this, currently hitting the same bug

Love django-ninja BTW, thanks a lot for all the work on it.

**Comment by vitalik** [comment]
@dhendry @abdelrahmenAyman 
fixed in 0.18
